src/executor/source/ApiInterface.ts

```diff
@@ -0,0 +1,13 @@
+import axios, { AxiosResponse } from 'axios'
+import https from 'https'
+
+const apiBaseAddress = process.env.API_URL
+
+export async function getApiVersion(): Promise<string> {
+  // This needs to be done since a self-signed certificate is used. This shoud NOT be used in production
+  const agent = new https.Agent( {
+    rejectUnauthorized: false
+  } )
```

Comment:
I'd at the very least like an issue with a `security`-tag or something like this before merging this.

Reply:
@S4nvers Just use the `process.env.NODE_ENV` to set this if the application is started in dev mode.

Reply:
Besides, the Ingress configuration on the cluster makes sure that only port 443 (HTTPS) can be used from the outside. The communication within the cluster network is using HTTP anyway.